packages/block-editor/src/components/date-format-picker/index.js

```diff
@@ -137,6 +137,7 @@ function NonDefaultControls( { format, onChange } ) {
 					label={ __( 'Custom format' ) }
 					hideLabelFromVision
 					help={ createInterpolateElement(
+						// translators: <Link> creates a link to the WordPress.org documentation on date and time formatting.
```

Comment:
Just a note for translators, so they don't change the link to a "valid" `<a ...>` element during the translation process.